src/libs/notification/example/example_notification.c

```diff
@@ -0,0 +1,105 @@
+/**
+ * @file
+ *
+ * @brief Implementation of notification functions as defined in kdbnotification.h
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <kdbassert.h>
+#include <kdbhelper.h>
+#include <kdbnotification.h>
+
+#include <signal.h>
+#include <stdio.h>
+#include <string.h>
+
+// from https://en.wikipedia.org/wiki/ANSI_escape_code#Colors
+#define ANSI_COLOR_RESET "\x1b[0m"
```

Comment:
Are they used in this example?

Reply:
Yes the are used to set the colors in `setTerminalColor()`.

Reply:
Okay, it was not clear to me how to use this then.